### PR TITLE
Layout for instant alert button

### PR DESCRIPTION
**Pull request by debugwand**
This change updates the buttons so that they lay out well on medium and large screens.  This is required because we have 4 cards together on these sized screens.

screen shot 2016-09-01 at 15 26 42
screen shot 2016-09-01 at 15 26 51
screen shot 2016-09-01 at 15 26 57

### CODE REVIEW

File: components/_concept.scss [hunk 1]
```diff
@@ -53,6 +53,13 @@
 	float: right;
 }
 
+.card__concept-instant {
+	@include respondTo(M, XL) {
```
Comment (keirog) [inline components/_concept.scss]: I'm curious about that second param. Is the intention for this to stop being applied for >XL? There is nothing larger at the moment

**Comment by keirog** [comment]
👍

File: components/_concept.scss (hunk 1 above)
Comment (debugwand) [inline components/_concept.scss]: the second argument is 'until', so everything above XL (1220px or higher) will not get these styles.

Comment (keirog) [inline components/_concept.scss]: ah sorry, I forgot the behaviour was "until & not-including _x_", rather than "until and including _x_" 😊

**Pull request merged**